### PR TITLE
Second

### PR DESCRIPTION
**Pull request by aliaksandrvorakh1990**

### CODE REVIEW

File: src/main/java/by/alex/vorakh/companies/controllers/CompanyServlet.java [hunk 1]
```diff
@@ -14,12 +14,12 @@
 @WebServlet("/companies")
 public class CompanyServlet extends AbstractController {
     private static final long serialVersionUID = 1L;
-    private static final CompanyService service= new CompanyService();
+    private static final CompanyService COMPANY_SERVICE = new CompanyService();
```
Comment (r-savko) [inline src/main/java/by/alex/vorakh/companies/controllers/CompanyServlet.java]: Bad practice to use static variables for non-constant variables. Please make it as non-static private fields.

File: src/main/java/by/alex/vorakh/companies/repository/CompanyDAO.java [hunk 2]
```diff
@@ -59,19 +59,19 @@ public Company get(int id) {
...
             connection = connectionPool.retrieve();
-            try (PreparedStatement preparedStatement = connection.prepareStatement(SELECT_MODEL_BY_ID);){
+            try (PreparedStatement preparedStatement = connection.prepareStatement(SELECT_COMPANY_BY_ID);){
                 preparedStatement.setInt(1, id);
                 resultSet = preparedStatement.executeQuery();
                 if(resultSet.next()){
                     company = new Company().setId(id).setName(resultSet.getString(1));
                 } else {
-                    throw new SQLException("The company does not exist"); // TODO add my exception for updating
+                    throw new DAOException (400, "The company does not exist");
```
Comment (r-savko) [inline src/main/java/by/alex/vorakh/companies/repository/CompanyDAO.java]: Please do not set an error code to the exceptions. Error code depends on the Exception class not from error code. DAO method are able to work not only on the web application.

Comment (r-savko) [inline src/main/java/by/alex/vorakh/companies/repository/CompanyDAO.java]: If a company was not found it does not mean that it is an exception. We just have no such company, it is valid response.

File: src/main/java/by/alex/vorakh/companies/controllers/CompanyServlet.java (hunk 1 above)
Comment (aliaksandrvorakh1990) [reply src/main/java/by/alex/vorakh/companies/controllers/CompanyServlet.java]: Solved

File: src/main/java/by/alex/vorakh/companies/repository/CompanyDAO.java (hunk 2 above)
Comment (aliaksandrvorakh1990) [reply src/main/java/by/alex/vorakh/companies/repository/CompanyDAO.java]: Solved

Comment (aliaksandrvorakh1990) [reply src/main/java/by/alex/vorakh/companies/repository/CompanyDAO.java]: Solved

**Pull request merged**